### ISSUE TITLE
Training with Cityscapes Dataset

### ISSUE THREAD

**Issue by TheDog223** [DrSleep/tensorflow-deeplab-resnet#198]
@DrSleep , @arslan-chaudhry I am trying to build my model using Cityscapes dataset. Because of the dataset's structure : 

https://github.com/mcordts/cityscapesScripts/blob/4031404ae3dffe901f45a91f7bb0253389284d6f/cityscapesscripts/helpers/labels.py 

some labels should be ignored in evaluation. How is this possible using your code? I can see that you ignore just 255 class. Is it possible to do this without changing the whole data-label preprocessing?

**Comment by DrSleep** [comment]
here you would need to change the condition of which labels to keep when calculating the loss: https://github.com/DrSleep/tensorflow-deeplab-resnet/blob/master/train.py#L170
